CodeGen/README.md

```diff
@@ -14,7 +14,7 @@ docker pull ghcr.io/huggingface/tgi-gaudi:1.2.1
 Alternatively, you can build the Docker image yourself with:
 
 ```bash
-bash ./serving/tgi_gaudi/build_docker.sh
```

Comment:
Why remove 'serving' directory?

Reply:
the `serving` directory doesn't exist

https://github.com/intel/GenAIExamples/blob/main/CodeGen/tgi_gaudi/build_docker.sh